src/components/progress-bar/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { CSSProperties } from 'react'
+import s from './progress-bar.module.css'
+
+/**
+ * Displays a bar that fills to the percent provided
+ * through the `percentDone` prop.
+ *
+ * `ProgressBar` is always `4px` tall, and grows to fill the width of its
+ * parent container. It also accepts an optional `rounded` boolean prop,
+ * which defaults to `true`. If `rounded` is set to `false`, the ends of
+ * the progress bar will end in sharp corners.
+ */
+function ProgressBar({
```

Comment:
`ProgressBar` is a presentational component needed for collection progress display in collection meta. Also planning to re-use it for other collection progress, and for the tutorial scroll indicator.